src/Card.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { FaUser } from "react-icons/fa";
+
+const Card = (props) => {
+  return (
+    <div className="card">
+      <div className="card-body">
+        <div className="img">
+          <FaUser size="5em" className="user-icon-card" />
+        </div>
+        <div className="card-info">
```

Comment:
You can destructure props right away in the argument of the Card component.

Reply:
Sure! Will make changes in the next PR. Thanks for the review!